modules/sdk-coin-near/src/near.ts

```diff
@@ -366,15 +375,21 @@ export class Near extends BaseCoin {
       const totalGasWithPadding = totalGasRequired.multipliedBy(1.5);
       const feeReserve = BigNumber(Networks[this.network].near.feeReserve);
       const storageReserve = BigNumber(Networks[this.network].near.storageReserve);
-      const netAmount = availableBalance.minus(totalGasWithPadding).minus(feeReserve).minus(storageReserve).toFixed();
+      const netAmount = availableBalance.minus(totalGasWithPadding).minus(feeReserve).minus(storageReserve);
+      if (netAmount.toNumber() <= 0) {
+        throw new Error(
+          `Found address ${i} with non-zero fund but fund is insufficient to support a recover ` +
```

Comment:
Maybe I'm misunderstanding the code, but wouldn't ${i} here only refer to the integer starting index? Or is this how you want it as opposed to the actual address/account id?

Reply:
Yep thats what we want. this is because WRW will eventually support letting the user specify which address index to start scanning from